src/odemis/driver/test/xt_client_test.py

```diff
@@ -609,7 +616,7 @@ def test_stigmator(self):
         self.assertEqual(self.microscope.get_stigmator()[0], stig_x)
         self.assertEqual(self.microscope.get_stigmator()[1], stig_y)
         # Try to set value outside of range
-        with self.assertRaises(OSError):
+        with self.assertRaises((OSError, ValueError)):
```

Comment:
I thought Pyro5 did not transfer ValueErrors, are you sure you need this?

I'm pretty sure the adapter only returns OSErrors. So if this is for the simulator, maybe we should update the simulator to always return OSErrors.

Reply:
Yes, that's for the simulator. I noticed that the current normal code either raise ValueError when the Python code checks for the bounds or OSError when it's XTLib/XTOM which checks. I concluded that both should be supported in the client, as it's hard to guess. So I've added it here, and decided to use ValueError in the simulator, as we check in the Python code, and didn't feel it's better to simulate a OSError. I can switch the simulator to OSError and here to just OSError if you see a good reason for it.

Reply:
Let's leave the ValueError then.